### PR TITLE
[16.0][MIG] foodhub_product_fields

### PR DESCRIPTION
**Pull request by victor-champonnois**
odoo task : https://gestion.coopiteasy.be/web#id=11999&action=475&active_id=568&model=project.task&view_type=form&menu_id=
(not deployed in test yet)

### CODE REVIEW

File: foodhub_product_fields/tests/test_product.py [hunk 1]
```diff
@@ -27,13 +26,7 @@ def setUpClass(cls):
...
                 "weight": 2,
                 "seller_ids": [
-                    (
-                        0,
-                        False,
-                        {
-                            "name": cls.supplier.id,
-                        },
-                    )
+                    (0, 0, {"partner_id": cls.supplier.id, "product_code": "ASUCODE"}),
```
Comment (robinkeunen) [inline foodhub_product_fields/tests/test_product.py]: `product_code` is not required, why did you add it ?

**Review by robinkeunen — APPROVED** [review]

File: foodhub_product_fields/tests/test_product.py (hunk 1 above)
Comment (victor-champonnois) [reply foodhub_product_fields/tests/test_product.py]: just a copy paste error, removed

**Comment by victor-champonnois** [comment]
rebased to up to date 16.0

**Review by polchampion — CHANGES_REQUESTED** [review]
@victor-champonnois Functional test feedback:
**Set Mandatory Fields in Product:** 
- [ ] Volume, Weight and Sale Price cannot be emptied in standard Odoo --> no need for this code in v16
- [x] Vendors - H.S. code - Sale Description - Website Product Category

**When duplicating a product, copy more fields over.**
- [ ] H.S. code and Country of origin are already copied automatically in standard odoo --> no code needed in v16
- [x] Volume - Weight - Vendors

**Comment by victor-champonnois** [comment]
> [ ] Volume, Weight and Sale Price cannot be emptied in standard Odoo --> no need for this code in v16

Why ? I checked and I see that these fields are prefilled with zero, which makes the required attribute not very effective, but this is also the case in v12.

**Comment by victor-champonnois** [comment]
> H.S. code and Country of origin are already copied automatically in standard odoo --> no code needed in v16

@polchampion done

> Volume, Weight and Sale Price cannot be emptied in standard Odoo --> no need for this code in v16

We decided to leave it like that

**Review by polchampion — APPROVED** [review]
